app.py

```diff
@@ -0,0 +1,20 @@
+# Application for webpage 
+
+from flask import Flask, render_template, redirect, url_for, request
+app = Flask(__name__)
+
+@app.route('/')
+def index():
+	return render_template('index.html')
+
+# Route for handling the login page logic
+@app.route('/login', methods=['GET', 'POST'])
+def login():
+	return render_template('login.html')
+	
+@app.route('/user/<name>')
+def user(name):
+  return render_template('user.html', name=name)
```

Comment:
the indentation level here is different than the rest of the file

Reply:
Done